hw/ip/flash_ctrl/data/flash_ctrl.hjson.tpl

```diff
@@ -1,6 +1,8 @@
 // Copyright lowRISC contributors.
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
+//
+## TODO: (uncomment) ${autogen_banner}
```

Comment:
why comment out this banner?

Reply:
This is unfortunately generated by a different `flash_ctrl_gen` script which does not yet understand what `autogen_banner` in the file is. I will address that in a separate commit once the other PR #8368 is merged.